Sources/SwiftSyntax/Documentation.docc/ChangingSwiftSyntax.md

```diff
@@ -6,74 +6,26 @@ Discover the steps necessary to add, remove, and update syntax nodes
 
 The surface area of the entire Swift programming language is represented in the
 Swift Syntax API and its associated data structures. In order to keep these
-structures in sync, this project makes use of a code generation utility called
-[gyb][GYB] - short for "Generate Your Boilerplate". 
-
-## Introduction to GYB
-
-The gyb utility allows for [literate metaprogramming][LiterateProgramming] of 
-files in any human-editable format. The standard convention is to attach the
-`.gyb` suffix to a file. For example, this repository uses `.swift.gyb` files
-that generate `.swift` code. The metaprogramming language that gyb uses is
-python. Templating code is delimited by `%` and can be used inline with `${...}`
-blocks. 
-
-Here is the gyb definition for all of the cases of the <doc:TokenKind> enum.
-
-```swift
-public enum TokenKind: Hashable {
-  case eof
-% for token in SYNTAX_TOKENS:
-%   kind = token.swift_kind()
-%
-%   # Tokens that don't have a set text have an associated value that
-%   # contains their text.
-%   if not token.text:
-%     kind += '(String)'
-%   end
-  case ${kind}
-% end
-}
-```
-
-- Note: gyb files act as the source of truth in this repository. Always edit
-        gyb files instead of the corresponding Swift files, or your changes will
-        be overwritten the next time those files are regenerated.
+structures in sync, this project makes generates the code using the Swift package 
```

Comment:
```suggestion
structures in sync, this project generates code using the Swift package 
```

Though it's not really clear to me what "API and structures" are being kept in sync due to generation.

Reply:
I think it’s just the definition of the syntax nodes. But let’s just keep that as-is for now.